src/index.ts

```diff
@@ -1,5 +1,5 @@
 import ApiEndpoints from './api-endpoints';
-import { logger } from './application-logger';
+import { logger as applicationLogger } from './application-logger';
```

Comment:
plays nicer with unused variable IDE warnings to rename this here than when exporting 🤷‍♂️